soco/services.py

```diff
@@ -161,6 +191,7 @@ def __init__(self, soco):
             611: 'Invalid Control URL',
             612: 'No Such Session',
         }
+        self.DEFAULT_ARGS = {}
```

Comment:
It's very similar to UPNP_ERRORS. Should I change the capitalization?

Reply:
The allcaps are by naming convention used for constants. EPNU_ERRORS is a constant, but since this is initialized to an empty dict, I assume it is not

Reply:
It _is_ constant as long as the object lives, but it's modified in the subclasses (e.g. [here](https://github.com/SoCo/SoCo/pull/573/files#diff-de75a829d8705a5dc2efa83021bffe7eR911)).
And we might want to add some defaults for convenience later.

Reply:
Ah. I missed that. Then I guess we are into the definition of what is considered a constant. My guess would be, that it is something that isn't changed after first defining it, since, in the languages that actually have constants, it isn't possible. Although they are only constants in the Python by naming convention, my guess would be that this definition applies and therefore DEFAULT_ARGS isn't

Reply:
Well, UPNP_ERRORS is [modified by subclasses](https://github.com/SoCo/SoCo/pull/573/files#diff-de75a829d8705a5dc2efa83021bffe7eR888) as well.

Reply:
Auch. Well, ok then. I stand by the definition above and think it is wrong for UPNP_ERRORS as well, but no reason to nit pick over that now. Let's go with it as is.